app/main.py

```diff
@@ -2,0 +1,109 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
+        pass
+
+    def player_info(self) -> None:
```

Comment:
The `player_info` method in the `Player` class is defined to return `None`, but it should return a `str` as per the implementations in subclasses. Consider changing the return type annotation to `str`.